geode-logging/build.gradle

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: "${rootDir}/${scriptDir}/standard-subproject-configuration.gradle"
+
+apply from: "${project.projectDir}/../gradle/publish.gradle"
+
+dependencies {
+    compile(platform(project(':boms:geode-all-bom')))
+
+    // Geode-common has annotations and other pieces used by geode-logging
+    api(project(':geode-common'))
+
+    implementation('org.apache.logging.log4j:log4j-api')
+
+    testCompile(project(':geode-junit')) {
+        exclude module: 'geode-logging'
+    }
+    testCompile(project(':geode-concurrency-test'))
+
+    testImplementation('com.tngtech.archunit:archunit-junit4')
+
+    testCompile('org.apache.bcel:bcel')
```

Comment:
Not sure this package needs this dependency

Reply:
i think ur correct